Our build framework sets PYTEST_PLUGINS to contain 'asyncio' and so the
	pytest_plugins = 'pytest_asyncio'
line tries to load the plugin 2nd time.  This emits additional warning and so
all tests that check the expected number of warnings fails here.

--- pytest_asyncio-1.4.0/tests/hypothesis/test_base.py.orig
+++ pytest_asyncio-1.4.0/tests/hypothesis/test_base.py
@@ -49,8 +49,6 @@
         from hypothesis import given
         import hypothesis.strategies as st
 
-        pytest_plugins = 'pytest_asyncio'
-
         @given(n=st.integers())
         async def test_hypothesis(n: int):
             assert isinstance(n, int)
@@ -68,8 +66,6 @@
         from hypothesis import given
         import hypothesis.strategies as st
 
-        pytest_plugins = 'pytest_asyncio'
-
         @given(n=st.integers())
         def test_hypothesis(request, n: int):
             markers = [marker.name for marker in request.node.own_markers]
--- pytest_asyncio-1.4.0/tests/test_event_loop_fixture.py.orig
+++ pytest_asyncio-1.4.0/tests/test_event_loop_fixture.py
@@ -13,8 +13,6 @@
             import asyncio
             import pytest
 
-            pytest_plugins = 'pytest_asyncio'
-
             @pytest.mark.asyncio
             async def test_something():
                 async def generator_fn():
@@ -36,7 +34,6 @@
             import asyncio
             import pytest
             import pytest_asyncio
-            pytest_plugins = 'pytest_asyncio'
 
             @pytest_asyncio.fixture
             async def _event_loop():
@@ -67,8 +64,6 @@
             import asyncio
             import pytest
 
-            pytest_plugins = 'pytest_asyncio'
-
             @pytest.mark.asyncio
             async def test_something():
                 # mock shutdown_asyncgen failure
--- pytest_asyncio-1.4.0/tests/test_set_event_loop.py.orig
+++ pytest_asyncio-1.4.0/tests/test_set_event_loop.py
@@ -112,8 +112,6 @@
             import pytest
             import pytest_asyncio
 
-            pytest_plugins = "pytest_asyncio"
-
             @pytest_asyncio.fixture
             async def async_fixture():
                 yield
